action.yml

```diff
@@ -0,0 +1,30 @@
+name: pysa
+description: "Run pysa on the repository and check for vulnerabilities"
+
+inputs:
+  build-type:
+    description: Pysa build to use. Possible values are stable and nightly. Default is stable.
+    required: false
+    default: "stable"
+
+runs:
+  using: "composite"
+  steps:
+    - run: if [ ${{inputs.build-type}} = "nighty" ]; then pip install pyre-check-nightly; else pip install pyre-check; fi
+      shell: bash
+    - run: if [ ! -f .pyre_configuration ]; then printf ".\n" | pyre init; fi
+      shell: bash
+    - run: pyre infer
+      shell: bash
+    - run: pyre infer -i --annotate-from-existing-stubs
+      shell: bash
+    - run: pip install fb-sapp
+      shell: bash
+    - run: mkdir ./.pysa/
+      shell: bash
+    - run: pyre analyze --no-verify --save-results-to ./
+      shell: bash
+    - run: sapp analyze ./taint-output.json
+      shell: bash
+    - run: sapp filter issues 1 ./.pysa/ --output-format sarif > sarif.json
```

Comment:
If the `./.pysa` directory here exists, I think the `./.pysa/` would be empty, resulting in no filters applied to the run.

The pysa filters are included with `pyre-check` package. It looks like in this case the filters are in the following directory: `/opt/hostedtoolcache/Python/3.9.6/x64/lib/pyre_check/pysa_filters`.

I recommend copying the setup instructions for Ubuntu from the [quickstart guide](https://pyre-check.org/docs/pysa-quickstart/) for this Github Action (e.g. use a virtual environment)

Reply:
Yeah I made a temporary directory as a last resort to see if giving an empty directory allows us to pass through. But it didn't and returned 0 issues. I will check the filters we have with pyre_check :)